tests/driver_ethos/Makefile

```diff
@@ -0,0 +1,7 @@
+# the radio driver to test
+USEMODULE = ethos
+
+CFLAGS += -DETHOS_UART=UART_DEV\(1\)
```

Comment:
```suggestion
# This test's purpose is to demonstrate the use of EthOS on a different serial,
# without running stdio over it; consequently, it can only be used on boards
# that actually have two UARTs.
CFLAGS += -DETHOS_UART=UART_DEV\(1\)
```

Reply:
let's better set those defaults in `ethos_params.h` the way slipdev does it.